### ISSUE TITLE
Missing field type for single checkbox

### ISSUE THREAD

**Issue by copleykj** [fermuch/simple-react-form-bootstrap#12]
Looks like there isn't a field type for a single checkbox, and the `CheckboxesField` doesn't allow for single options with a Boolean type.

**Comment by fermuch** [comment]
Hey!

Right now I'm on vacation, but on my spare time I'm working on a rework of SRF Bootstrap, since I see a lot more of usage coming to SRF.
The codebase right now is in bad state, because i started the project for personal usage, but I'm working on a local branch right now and I'd like to know what use cases you have and what are your current pain-points using the library, apart from the total lack of docs. 

TL;DR: What would you like to see in SRF bootstrap?

**Comment by copleykj** [comment]
Hey, thanks for getting back so fast while on vacation.

At this point it's fitting most of my use cases with only a couple minor issues that I'm working around. 

First one is the lack of the single check box. I have a single field that just needs to be true or false depending on if the box is checked or not.

The second is the select boxes. It would be nice if they behaved like what you would expect from a standard html from. By this I mean that if there is a value provided to the component either through the value attribute manually or through props passed down from the form component via the forms doc prop, then the select attribute should be set on the proper `<option>` tag. If there isn't a value supplied by the doc or value prop, then the value of the first option should be passed up to the form via the onChange prop. Also it might be nice to have a way to specify option groups, but that's not that important to me at this point.

If I find anything else I'll be sure to file issues on them.
